src/ncp/ncp_base.cpp

```diff
@@ -279,6 +279,7 @@ NcpBase::NcpBase(Instance *aInstance)
 #if OPENTHREAD_CONFIG_ENABLE_STEERING_DATA_SET_OOB
     memset(&mSteeringDataAddress, 0, sizeof(mSteeringDataAddress));
 #endif
+    otThreadRegisterParentResponseCallback(mInstance, &NcpBase::HandleParentResponseInfo, static_cast<void *>(this));
```

Comment:
otSetParentResponseCallback(... &NcpBase::HandleParentResponseReceived ...);

(Set is used instead of Register for other similar fns in this file)

Reply:
Seems best to be consistent. I'll defer to @abtink / @jwhui.